colossalai/zero/gemini/gemini_ddp.py

```diff
@@ -318,6 +331,7 @@ def backward(self, loss: torch.Tensor):
         self._post_backward()
 
     def backward_by_grad(self, tensor, grad):
+        raise RuntimeError("Gemini is not compatible with pipeline. backward_by_grad shoudn't be called in Gemini.")
```

Comment:
The lines below this line should be deleted

Reply:
updated